### DISCUSSION TITLE
New features in the recent preview versions

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#379] [Announcements]
I have implemented a couple of new features I would like to announce here explicitly.

In the last preview version 1.10.3_03 there is a new switch "remove icon if exists" at the icon tool bar and also available in the menu and in the preferences
![Screenshot at 2022-05-18 20-11-08-cropped](https://user-images.githubusercontent.com/1558052/169116058-1f633557-be0f-4926-9282-9babd11b7380.png)
If it is enabled, clicking on any icon already present at the node removes this icon instead of adding it again.

Because I wanted to implement it fast I have created the icon myself, I think our artists could improve it. Also if you think the description text could be changed, let me know.

Since version 1.10.3_02 there is also a new layout option View->View settings->Allow compact layout. It can be activated per map and saved in a map template. If it is enabled, moving nodes up allows to overlap them with any nodes above making creating user specific layouts easier than using the free positioned nodes.

**Comment by ldmpub** [comment]
Thanks for this feature announcement, it will help me for Freeplane translation 👍.
